### PR TITLE
kafka logs added

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: module/observationSubmissions/helper.js [hunk 1]
```diff
@@ -123,11 +123,15 @@ module.exports = class ObservationSubmissionsHelper {
...
+                console.log("--------------------------OBS WITH RUBRIC ISSUE DEGUG START pushObservationSubmissionForReporting()-------------------");
 
-               
+                console.log("STATUS : ",observationSubmissionsDocument.status);
+                console.log("OBS SUBMISSION ID : ",observationSubmissionId);
 
+                console.log("BEFORE CALLING pushObservationSubmissionToKafka method ");
                 const kafkaMessage = await kafkaClient.pushObservationSubmissionToKafka(observationSubmissionsDocument);
-               
+                console.log("AFTER CALLING pushObservationSubmissionToKafka method ");
```
Comment (aks30) [inline module/observationSubmissions/helper.js]: @rakeshSgr Log the kafka message also here

Comment (aks30) [reply module/observationSubmissions/helper.js]: Use JSON stringify

File: module/observationSubmissions/helper.js [hunk 2]
```diff
@@ -330,15 +341,21 @@ module.exports = class ObservationSubmissionsHelper {
...
                     return resolve(messageConstants.apiResponses.OBSERVATION_RATING);
                 } else {
                     emailClient.pushMailToEmailService(emailRecipients,messageConstants.apiResponses.OBSERVATION_AUTO_RATING_FAILED+" - "+submissionId,JSON.stringify(resultingArray));
                     return resolve(messageConstants.apiResponses.OBSERVATION_RATING);
                 }
 
+                
+
             } catch (error) {
+                console.log("rateSubmissionById() catch block",error);
```
Comment (aks30) [inline module/observationSubmissions/helper.js]: @rakeshSgr this and error above log it in diff line and use JSON stringify

Comment (rakeshSgr) [reply module/observationSubmissions/helper.js]: updated

**Pull request merged**